canonical/snapcraft.io
# Convert snap settings page to React

## Done
Converted the snap settings page to React

## How to QA
- Go to https://snapcraft-io-4088.demos.haus/SNAP_NAME/settings?show_react_settings=true
- Check that the form works as expected
